webpack/components/extensions/Hosts/BulkActions/BulkErrataWizard/04_Review.js

```diff
@@ -0,0 +1,160 @@
+import React, { useContext, useState } from 'react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { FormattedMessage } from 'react-intl';
+import { TreeView, Button, Text, TextContent, TextVariants, Flex, FlexItem, Dropdown, DropdownItem, DropdownToggle } from '@patternfly/react-core';
+import { useWizardContext } from '@patternfly/react-core/next';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { BulkErrataWizardContext } from './BulkErrataWizard';
+
+export const dropdownOptions = [
+  __('via remote execution'),
+  __('via customized remote execution'),
+];
+
+export const BulkErrataReview = () => {
+  const { goToStepById } = useWizardContext();
+  const {
+    finishButtonText,
+    selectedRexOption,
+    setSelectedRexOption,
+    finishButtonLoading,
+    errataBulkSelect: {
+      selectedResults: selectedErrataResults,
+      selectedCount: currentSelectedErrataCount,
+    },
+    hostsBulkSelect: {
+      selectedCount: currentSelectedHostsCount,
+    },
+  } = useContext(BulkErrataWizardContext);
+
+  const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+  const toggleDropdownOpen = () => setIsDropdownOpen(prev => !prev);
+  const handleSelect = () => {
+    setIsDropdownOpen(false);
+  };
+
+  const dropdownItems = dropdownOptions.map(text => (
+    <DropdownItem key={`option_${text}`} ouiaId={`option_${text}`} onClick={() => setSelectedRexOption(text)}>{text}</DropdownItem>
+  ));
+
+  const treeViewTitle = __('Errata to apply');
+  const treeViewData = [
+    {
+      name: treeViewTitle,
+      id: 'errata-treeview-title',
+      customBadgeContent: currentSelectedErrataCount,
+      children:
+        selectedErrataResults.map(({ id, name }) => ({
+          name: `${id}: ${name}`,
+          id,
+          key: id,
+        })),
+      action: (
+        <Button
+          ouiaId="link-to-errata-wizard-step-2"
+          variant="link"
+          type="button"
+          aria-label="Edit errata list"
+          onClick={() => goToStepById('mew-step-1')}
+        >
+          {__('Edit')}
+        </Button>
+      ),
+      actionProps: {
+        'aria-label': 'Edit errata list',
+      },
+    },
+  ];
+
+  const hostTreeViewData = [
+    {
+      name: __('Hosts'),
+      id: 'errata-host-treeview-title',
+      customBadgeContent: currentSelectedHostsCount,
+      expandedIcon: null,
+      action: (
+        <Button
+          ouiaId="link-to-errata-wizard-step-3"
+          variant="link"
+          type="button"
+          aria-label="Edit host selection"
+          onClick={() => goToStepById('mpw-step-3')}
```

Comment:
```suggestion
          onClick={() => goToStepById('mew-step-3')}
```

I also see 6 more places `mpw` needs to change to `mew`